ledger-service/http-json/src/failurelib/scala/http/ToxicSandboxFixture.scala

```diff
@@ -4,36 +4,44 @@
 package com.daml.http
 
 import com.daml.bazeltools.BazelRunfiles
-import com.daml.ledger.api.testing.utils.{OwnedResource, SuiteResource, Resource => TestResource}
+import com.daml.ledger.api.domain.LedgerId
 import com.daml.ledger.resources.{Resource, ResourceContext, ResourceOwner}
-import com.daml.ledger.sandbox.SandboxOnXForTest.{ConfigAdaptor, dataSource}
-import com.daml.ledger.sandbox.SandboxOnXRunner
+import com.daml.lf.integrationtest.CantonFixtureWithResource
 import com.daml.platform.apiserver.services.GrpcClientResource
-import com.daml.platform.sandbox.{
-  AbstractSandboxFixture,
-  SandboxBackend,
-  SandboxRequiringAuthorizationFuns,
-  UploadPackageHelper,
-}
 import com.daml.ports.{LockedFreePort, Port}
 import com.daml.timer.RetryStrategy
 import eu.rekawek.toxiproxy._
 import io.grpc.Channel
 import org.scalatest.{BeforeAndAfterEach, Suite}
 
+import java.io.File
 import java.net.InetAddress
+import java.nio.file.Path
 import scala.concurrent.Future
 import scala.concurrent.duration.DurationInt
 import scala.sys.process.Process
 
-// Fixture for Sandbox Next behind toxiproxy to simulate failures.
+// Fixture for Canton behind toxiproxy to simulate failures.
 trait ToxicSandboxFixture
```

Comment:
@carlpulley-da should we factorize this one with the one from script ?

Reply:
Sure okay - lets do this in a separate PR, I'll make a note

Reply:
@samuel-williams-da thanks, much appreciated.

Reply:
#16834